src/components/Header/Header.js

```diff
@@ -0,0 +1,45 @@
+import './Header.css';
+
+import { Link } from 'react-router-dom';
+
+const Header = () => (
+  <header>
+    <div>
+      <option>
+        <Link to="Sign In">Sign In</Link>
+      </option>
+      <option>
+        <Link to="Sign Up">Sign Up</Link>
+      </option>
+      <option>
+        <Link to="Log Out">Log Out</Link>
+      </option>
+    </div>
+    <nav id="nav3">
+      <h1>OSS-Project</h1>
+      <ul>
+        <li>
+          <Link to="/popular">Popular</Link>
+        </li>
+        <li>
+          <Link to="/animation">Animation</Link>
+        </li>
+        <li>
+          <Link to="/romance">Romance</Link>
+        </li>
+        <li>
+          <Link to="/science">Science Fiction</Link>
+        </li>
+        <li>
+          <Link to="/action">Action</Link>
+        </li>
```

Comment:
이부분은 하드코딩하기 보다 list, map 함수를 이용하는게 좋아 보여요

```javascript
const categoryList = ['Popular', 'Animation', 'Romance', 'Science Fiction', 'Action'];
```

Header 컴포넌트 밖에 위와 같이 list로 정의합니다.

```suggestion
        {categoryList.map((list) => (
          <li key={list}>
            <Link to={list.toLowerCase()}>{list}</Link>
          </li>
        ))}
```